Source/Scene/Material.js

```diff
@@ -226,6 +232,23 @@ define([
      *      <li><code>outlineColor</code>: diffuse color and alpha for the outline.</li>
      *      <li><code>outlineWidth</code>: width of the outline in pixels.</li>
      *  </ul>
+     *  <li>ElevationContour</li>
+     *  <ul>
+     *      <li><code>color</code>: color and alpha for the contour line.</li>
+     *      <li><code>spacing</code>: spacing for contour lines in meters.</li>
+     *      <li><code>lineThickness</code>: Number specifying the thickness of the grid lines in pixels.</li>
```

Comment:
To be consistent with the rest of the Cesium API, can this be "width" instead of "thickness?"